src/magnify/registry.py

```diff
@@ -65,11 +571,260 @@ def ps_chip(**kwargs):
     return pipe
 
 
-def pc_chip(**kwargs):
+def pc_chip(
+    data: ArrayLike | str,
+    times: Sequence[int] | None = None,
+    channels: Sequence[str] | None = None,
+    debug: bool = False,
+    shape: tuple[int, int] | None = None,
+    pinlist: str | None = None,
+    blank: str | list[str] | None = None,
+    overlap: int = 102,
+    rotation: int | float = 0,
+    row_dist: float = 406 / 3.22,
+    col_dist: float = 750 / 3.22,
+    min_button_radius: int = 4,
+    max_button_radius: int = 15,
+    low_edge_quantile: float = 0.1,
+    high_edge_quantile: float = 0.9,
+    num_iter: int = 5000000,
+    min_roundness: float = 0.2,
+    cluster_penalty: float = 50,
+    roi_length: int = 61,
+    progress_bar: bool = False,
+    search_timestep: list[int] | None = None,
+    search_channel: str | list[str] | None = None,
+    squeeze: bool = True,
+    roi_only: bool = False,
+    drop_tiles: bool = True,
+) -> xr.Dataset | list[xr.Dataset]:
+    pipe = pc_chip_pipe(
+        debug=debug,
+        shape=shape,
+        pinlist=pinlist,
+        blank=blank,
+        overlap=overlap,
+        rotation=rotation,
+        row_dist=row_dist,
+        col_dist=col_dist,
+        min_button_radius=min_button_radius,
+        max_button_radius=max_button_radius,
+        low_edge_quantile=low_edge_quantile,
+        high_edge_quantile=high_edge_quantile,
+        num_iter=num_iter,
+        min_roundness=min_roundness,
+        cluster_penalty=cluster_penalty,
+        roi_length=roi_length,
+        progress_bar=progress_bar,
+        search_timestep=search_timestep,
+        search_channel=search_channel,
+        squeeze=squeeze,
+        roi_only=roi_only,
+        drop_tiles=drop_tiles,
+    )
+    return pipe(data=data, times=times, channels=channels)
+
+
+"""
+    Processes image data to detect buttons on a PC (Polycarbonate) chip, applying stitching, rotation, horizontal flips, and multiple filtering steps.
```

Comment:
Not a polycarbonate chip, once again named after the designer but you can just leave it as an acronym.

Reply:
Resolved.